packages/api/cms-api/src/dam/files/files.resolver.ts

```diff
@@ -151,7 +156,78 @@ export function createFilesResolver({ File, Scope: PassedScope }: { File: Type<F
             })
             inboxFolderId: string,
         ): Promise<CopyFilesResponseInterface> {
-            return this.filesService.copyFilesToScope({ fileIds, inboxFolderId, user });
+            return this.copyFiles(user, fileIds, inboxFolderId);
+        }
+
+        @Mutation(() => CopyFilesResponse)
+        @SkipBuild()
+        async copyFiles(
+            @GetCurrentUser() user: CurrentUserInterface,
+            @Args("fileIds", { type: () => [ID] }) fileIds: string[],
+            @Args("targetFolderId", {
+                type: () => ID,
+                nullable: true,
+            })
+            targetFolderId: string | undefined,
+            @Args("targetScope", {
+                type: () => Scope,
+                nullable: true,
+                defaultValue: hasNonEmptyScope ? undefined : {},
+                description: "Doesn't need to be set if targetFolderId is set",
+            })
+            targetScope?: typeof Scope,
```

Comment:
`copyFiles` is a better name than `copyFilesToScope` IMO

I also added the new possiblity to copy to a root folder by setting the `targetFolderId` to `undefined` and `targetScope` to the target scope (or undefined if no DAM scoping is used)